[10.x.x] Fixed PostFX MotionBlur effect not working in XR (#1761)
Unity-Technologies/Graphics: com.unity.render-pipelines.universal/CHANGELOG.md

- Fixed issue that caused color grading to not work correctly with camera stacking. [case 1263193](https://issuetracker.unity3d.com/product/unity/issues/guid/1263193/)
- Fixed an issue that caused an infinite asset database reimport when running Unity in command line with -testResults argument.
- Fixed issue where blur would sometimes flicker [case 1224915](https://issuetracker.unity3d.com/issues/urp-bloom-effect-flickers-when-using-integrated-post-processing-feature-set)

> - Fixed issue where motion blur pass not working in XR.

Reviewer: ```suggestion
- Fixed issue that caused motion blur to not work in XR.```
